common/lib/dependabot/notifications.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require "active_support"
```

Comment:
Per a [comment](https://github.com/rails/rails/issues/43889#issuecomment-995364395) on rails/rails, 

> Requiring individual files from Active Support without first requiring the top-level active_support is not supported.

This played out in the test suite for the previous commit, with errors of the following type:

```plaintext
      Failure/Error:
        Dependabot.instrument(
          Notifications::FILE_PARSER_PACKAGE_MANAGER_VERSION_PARSED,
          ecosystem: "bundler",
          package_managers: {
            "bundler" => version
          }
        )

      NameError:
        uninitialized constant ActiveSupport::IsolatedExecutionState
```

Reply:
Nice find!